src/server/game/Entities/Player/Player.h

```diff
@@ -2959,6 +2959,8 @@ class Player : public Unit, public GridObject<Player>
 
     Optional<float> _farSightDistance = { };
 
+    bool _wasOutdoor;
+    
```

Comment:
remove the blank space on line 2963.
![image](https://user-images.githubusercontent.com/16887899/149986116-a12f59c7-ad4c-41a2-8dbc-54ac0d49a776.png)